### PR TITLE
BEDS 991/new clickhouse schema v2.5.2 changes

### PR DESCRIPTION
**Pull request by sasha-bitfly**

### CODE REVIEW

**Review by invis-bitfly — CHANGES_REQUESTED** [review]
Hi, please update [this](https://github.com/gobitfly/beaconchain/blob/3e1a8fd4a807331824a36e79bf31d0127f82abc1/backend/pkg/api/data_access/vdb_blocks.go#L360-L394) to remove the holesky specific branch and the mention of mainnet in the clickhouse query

File: backend/pkg/api/data_access/vdb_summary.go [hunk 1]
```diff
@@ -1183,16 +1196,16 @@ func (d *DataAccessService) GetLatestExportedChartTs(ctx context.Context, aggreg
...
+		table = "validator_dashboard_data_hourly_max_ts"
 		dateColumn = "t"
 	case enums.IntervalDaily:
-		table = "view_validator_dashboard_data_daily_max_ts"
+		table = "validator_dashboard_data_daily_max_ts"
 		dateColumn = "t"
 	case enums.IntervalWeekly:
-		table = "view_validator_dashboard_data_weekly_max_ts"
+		table = "validator_dashboard_data_weekly_max_ts"
 		dateColumn = "t"
```
Comment (invis-bitfly) [inline backend/pkg/api/data_access/vdb_summary.go]: @sasha-bitfly please revert these changes again, the views have been renamed in the schema to be `view_` prefixed again

Comment (sasha-bitfly) [reply backend/pkg/api/data_access/vdb_summary.go]: @invis-bitfly [Done](https://github.com/gobitfly/beaconchain/commit/84f1b0431860855be156446ac6335e034be5786c)

File: backend/pkg/api/data_access/vdb_blocks.go [hunk 2]
```diff
@@ -361,37 +361,22 @@ func (d *DataAccessService) GetValidatorDashboardBlocks(ctx context.Context, das
...
-		if err != nil {
-			return nil, nil, err
-		}
-		err = d.clickhouseReader.SelectContext(ctx, &clRewardsData, clRewardsQuerySql, args...)
-		if err != nil {
-			return nil, nil, err
-		}
+
+	clRewardsQuery := goqu.Dialect("postgres").
+		From(goqu.L("validator_proposal_rewards_slot")).
```
Comment (remoterami) [inline backend/pkg/api/data_access/vdb_blocks.go:366]: > Hi, please update [this](https://github.com/gobitfly/beaconchain/blob/3e1a8fd4a807331824a36e79bf31d0127f82abc1/backend/pkg/api/data_access/vdb_blocks.go#L360-L394) to remove the holesky specific branch and the mention of mainnet in the clickhouse query

That table doesn't seem to exist yet on clickhouse for testnets. @invis-bitfly Are there missing migrations files or something? Otherwise we might need to keep using alloy data for testnets

Edit: nvm, was a db access permission issue
